backend/app/services/context/context_service.py

```
@@ -237,11 +267,92 @@ def upload_attachment(
+        # Update status to PARSING
+        context.status = ContextStatus.PARSING.value
+        db.flush()
+
+        # Parse document
+        try:
+            truncation_info = self._parse_and_update_context(
+                context=context,
+                binary_data=binary_data,
+                extension=extension,
+            )
+        except DocumentParseError as e:
```

Ruff (0.14.14)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)